coala_quickstart/generation/Settings.py

```diff
@@ -61,7 +61,7 @@ def generate_ignore_field(project_dir, languages, extset, ignore_globs):
     for glob in ignore_globs:
         gitignore_files = {file
                            for file in collect_files([glob], null_printer)}
-        if not all_files.isdisjoint(gitignore_files):
+        if all_files.isdisjoint(gitignore_files):
```

Comment:
hmm, `all_files` should already be ignoring the `ignore_globs` but I don't think it's happening currently (seems like a bug `collect_files` function). 

Also, I thought over the logic and we shouldn't really have this disjoint logic because "If there are no file in `all_files` which is ignored in `.gitignore`, then the `gitignore_files` should be empty because no such files exist"